model/src/main/java/org/cloudfoundry/identity/uaa/zone/TokenPolicy.java

```diff
@@ -116,6 +117,14 @@ public void setRefreshTokenUnique(boolean refreshTokenUnique) {
         this.refreshTokenUnique = refreshTokenUnique;
     }
 
+    public boolean isRefreshTokenRotate() {
```

Comment:
The name of this getter is awkward to understand to me. What about something like `shouldRotateRefreshTokens()` ?

Reply:
TokenPolicy is used to read JSON data and here is<variable> is need, but I renamed the attribute from RefreshTokenCreator